source/_cookbook/track_battery_level.markdown

```diff
@@ -29,7 +29,30 @@ sensor:
 ```
 {% endraw %}
 
+### Android Devices
+
```

Comment:
Maybe add some extra information about the Home Assistant android app etc. Just how it is on the iOS devices